aidants_connect_web/models.py

```diff
@@ -1328,6 +1328,9 @@ class DatavizDepartment(models.Model):
         "Nom de département", max_length=50, null=False, blank=False
     )
 
+    def normalize_zipcode(self):
+        return "".join([x if x.isdigit() else "0" for x in self.zipcode])
```

Comment:
Curiosité Python : est-ce que ça marche ici de supprimer les [] pour faire une seule itération de la chaîne ? Ou bien il faudrait de toute façon les changer en () ?